edx_rest_api_client/tests/test_client.py

```diff
@@ -143,7 +55,6 @@ def test_get_client_credential_access_token_failure(self, code, body):
         """
         with self.assertRaises(requests.RequestException):
             self._mock_auth_api(OAUTH_URL, code, body=body)
-            EdxRestApiClient.get_oauth_access_token(OAUTH_URL, "client_id", "client_secret")
```

Comment:
This causes this test to fail, because the `self.assertRaises` expected an error to happen in `EdxRestApiClient`.